docs/common/decorators/tags.md

```diff
@@ -0,0 +1,19 @@
+All Rugs should be annotated with one or more `#!typescript @Tags` 
+[decorators][decorators] to optimize their discoverability. For 
+example if you were to create an Rug that alters a README file
+then the following `#!typescript @Tags` would be applicable:
+
+```typescript
+@Tags("readme", "documentation")
+```
+
+Tag values should consist of only lower case letters, numbers, and
+dashes (`-`).
+
+Please try to ensure at least one of the tags on your Rug maps to an image
```

Comment:
Rather than "please try to ensure" could say "If possible try to include" as it may be the case than none are appropriate.